tests/osaka/eip7692_eof_v1/eip4750_functions/test_code_validation.py

```diff
@@ -572,6 +572,77 @@ def test_callf_stack_overflow_by_height(eof_test, callee_stack_height):
     eof_test(container=container, expect_exception=EOFException.STACK_OVERFLOW)
 
 
+@pytest.mark.parametrize(
+    "container",
+    [
+        Container(
+            name="underflow_1",
+            sections=[
+                Section.Code(
+                    code=Op.CALLF[1] + Op.STOP,
+                    max_stack_height=1,
```

Comment:
I would add a duplicate to this case but with `max_stack_height=2`, just for it to match the `code_outputs=2` of the second section.

Reply:
Added as "underflow_2".